### PR TITLE
VP-3862: Change document structure according to mkdocs requirements

### PR DESCRIPTION
**Pull request by mayaalsamarrai**
### Proposed of changes
Describe the technical details of realization provided by you.

### Make sure these boxes are checked:
- [ ] Check all the changes in github PR - files count (non of them are redundant, have meaningful changes, all are added), if target branch is correct
- [ ] Check methods and variable namings - it should be self descriptive, no typos
- [ ] Check you did not introduce breaking changes in API and public models/services.
- [ ] Respect extensibility - https://community.virtocommerce.com/t/extensibility-basics-the-domain-model-and-persistence-layer-extension/141
- [ ] Follow [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and [SOLID](https://en.wikipedia.org/wiki/SOLID) principles
- [ ] For unit tests - follow Microsoft best practices: https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices
- [ ] Consolidate solution dependencies in case you are using newer version, update VC module dependencies in module.manifest. Do not upgrade 3rd party packages that are shipped with the platform with the version newer than in the platform.
- [ ] Check code style conventions - https://github.com/VirtoCommerce/styleguide/blob/master/csharp.md
- [ ] Check PR have a concise and descriptive title, follow [git commit message rules](https://github.com/VirtoCommerce/styleguide/blob/master/gitcommits.md)

### CODE REVIEW

**Comment by lnetrebskii** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-2883 has been created

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+1. Tax providers registration using the code;
+1. Display the list of available tax providers on UI;
+1. Edit tax provider settings on UI;
+1. Connect the tax providers to the selected Store;
+
+## Scenarios
+
+### View Tax Provider
+
+1. Go to More->Stores->select the Store;
```
Comment (yecli) [inline README.md:17]: I would remove Avalara at all and used Fixed tax provider as an example. As it is the one that is shipped with the module. And need to change the screenshosts

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+## Overview
+
+The tax module is an engine for tax calculations and allows doing the following:
+
+1. Extend the custom tax providers;
+1. Tax providers registration using the code;
+1. Display the list of available tax providers on UI;
+1. Edit tax provider settings on UI;
+1. Connect the tax providers to the selected Store;
+
```
Comment (yecli) [inline README.md:12]: Please add this line:
API to work with tax provider list;

File: README.md [hunk 3]
```diff
@@ -1 +1,42 @@
...
+![Tax providers](docs/media/screen-tax-providers.png)
+![Tax providers details](docs/media/screen-tax-provider-details.png)
+
+### Edit Tax Provider Settings
+
+1. Select the 'Settings' widget on tax provider details blade;
+1. The following default settings can be edited on UI if needed:
+
+     1. Company code in Avalara system;
```
Comment (yecli) [inline README.md:37]: I think these steps should be aligned with Fixed Rate provider. Now they are Avalara specific.

**Pull request merged**